Userland/Libraries/LibWasm/AbstractMachine/Operators.h

```diff
@@ -710,6 +710,23 @@ struct VectorIntegerBinaryOp {
     }
 };
 
+template<size_t VectorSize>
+struct VectorBitmask {
+    auto operator()(u128 lhs) const
+    {
+        using VectorType = NativeVectorType<128 / VectorSize, VectorSize, MakeSigned>;
+        auto value = bit_cast<VectorType>(lhs);
+        u32 result = 0;
+
+        for (size_t i = 0; i < VectorSize; ++i)
+            result |= (value[i] < 0) << i;
```

Comment:
hmm...an explicit `u32` static cast for the nice trick here would be appreciated.
```suggestion
            result |= static_cast<u32>(value[i] < 0) << i;
```